source/index.html.md.erb

```diff
@@ -37,18 +37,47 @@ DfE Apply will avoid prioprietary codes wherever possible, preferring existing d
 
 Codes appear in three contexts:
 
-- All dates in in the API specification are intended to be [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) compliant
-- Nationality is expressed as an [ISO 3166](https://www.iso.org/iso-3166-country-codes.html) country code
-- Disability is expressed as a [HESA Disability type code](https://www.hesa.ac.uk/collection/student/datafutures/a/disability_disability)
+- All dates in in the API specification are intended to be ISO 8601 compliant
+- Nationality is expressed as an ISO 3611 country code
 
 ## How do I connect to this API?
 
-**This is a working draft specification**.
-
-We are currently building the service. The API has not yet been implemented.
-
-We're working with all vendors on timelines for integration. As part of this, we'll make private testing environments available.
-
-If you need information about what we're working on now and next, [contact us](mailto:becomingateacher@digital.education.gov.uk) at becomingateacher@digital.education.gov.uk.
+### Authentication and authorisation
+
+The data held by the Apply service is confidential, only available
+to candidates themselves and staff from the training provider to
+which applications are made. Therefore authentication will be required
+for all API interactions.
+
+Training Provider Staff identify themselves to Apply using DfE
+Sign-in, the Single Sign-on service used on the existing _Find
```

Comment:
probably should refer to `Publish teacher training courses`:

![image](https://user-images.githubusercontent.com/23801/64708015-cfa98400-d4ab-11e9-8e93-e14a3f8d5f30.png)